sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_models.py

```diff
@@ -168,7 +173,13 @@ def __init__(self, feature_id, enabled, filters=[], **kwargs):  # pylint: disabl
         self.etag = kwargs.get("etag", None)
         self.description = kwargs.get("description", None)
         self.display_name = kwargs.get("display_name", None)
-        self.value = kwargs.get("value", {"enabled": enabled, "conditions": {"client_filters": filters}})
+        if "enabled" in kwargs.keys():
+            self.value = kwargs.get(
```

Comment:
Given FeatureFlagConfigurationSetting is a sub-class of ConfigurationSetting, I would expect FeatureFlagConfigurationSetting.value has same meaning of ConfigurationSetting.value which is a string.

Why we want to expose "value" to the user as a dict?

Reply:
The `value` returned from the service is a json object serialized into a string. Deserializing into a dict was the design you and I came up with in the last sprint.

Reply:
Maybe I missed that. Then we need a getter for value to return string. We need to be consistent that ConfigSetting.value has same meaning of sub-class.value.